File: Framework.D-2015/Framework.D-2015/Cache/CacheUsuario.vb
Namespace Cache
    ''' <summary>
    ''' Clase estatica para guardar los datos del usuario de forma temporal en memoria,
    ''' una vez se que la aplicacion se cierra no guarda los datos
    ''' </summary>
    Public Class CacheUsuario

        Public Shared iduser As Integer

        Public Shared username As String

        Public Shared password As String

        Public Shared email As String

        Public Shared rol As String

    End Class
End Namespace

File: Framework.D-2015/Framework.D-2015/Cache/CacheUsuario.cs

namespace Framework.D_2015.Cache
{
    /// <summary>
    /// Clase estatica para guardar los datos del usuario de forma temporal en memoria,
    /// una vez se que la aplicacion se cierra no guarda los datos
    /// </summary>
    public class CacheUsuario
    {
        public static int iduser;
        public static string username;
        public static string password;
        public static string email;
        public static string rol;
    }
}